carservice/models.py

```diff
@@ -0,0 +1,57 @@
+from django.db import models
+from django.core.validators import MinValueValidator
+from django.contrib.auth.models import User
+from django.utils import timezone
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext_lazy as _
+
+
+def validate_ser_num(value):
+    if len(value) != 17:
+        raise ValidationError(
+            _('Please, put the correct VIN number.'),
+            params={"value": value},
+        )
+
+
+class Car(models.Model):
+    serial_number = models.CharField(max_length=17, validators=[validate_ser_num])
+    car_mileage = models.PositiveIntegerField()
+    car_model = models.CharField(max_length=30)
+    car_year = models.DateField()
+
+    def __str__(self):
+        return f'Model: {self.car_model}, serial number:({self.serial_number})'
+
+
+class Offer(models.Model):
+    description = models.TextField()
+    price = models.FloatField(validators=[MinValueValidator(10.0)])
+
+    car = models.ForeignKey(Car, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return f'Price: ({self.price}), car:({self.car})'
+
+
+class Rent(models.Model):
+    status = models.CharField(max_length=30, blank=True, null=True)
```

Comment:
Ok, a wiemy jakie te statusy powinny być, bo raczej to nie powinien być jakiś tekst randomowy zawsze inny, tylko coś określonego, zobacz na coś takiego jak `choices` w tym kontekście